wdl-lint/src/rules/import_sort.rs

```diff
@@ -36,6 +39,11 @@ fn improper_comment(span: Span) -> Diagnostic {
         .with_fix("remove the comment from the import statement")
 }
 
+/// Removes extra whitespace from a string by replacing multiple spaces with a
+/// single space.
+fn normalize(s: &str) -> String {
+    s.split_whitespace().collect::<Vec<_>>().join(" ")
+}
```

Comment:
```suggestion
```
as discussed, this will be handled by `wdl-format` in the future, so we don't need to do any normalizing here.